src/Validator.php

```diff
@@ -29,7 +29,7 @@ public static function validateClassName(string $className, string $errorMessage
         $pieces = explode('\\', ltrim($className, '\\'));
 
         foreach ($pieces as $piece) {
-            if (!preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $piece)) {
+            if (!Str::isValidPhpClassName($piece)) {
```

Comment:
Maybe will be better not to change this check, but add a new check with new error message, to show the user what is wrong with classname?